source/industry-solutions.txt

```diff
@@ -3,3 +3,8 @@
 ==================
 Industry Solutions
 ==================
+
+.. toctree::
```

Comment:
Not sure if we want to be doing this, but I added a definition list entry for my article to the main body of this page, just using the description ([GitHub](https://github.com/mongodb/docs-atlas-architecture/pull/166/files#diff-9b6286d24333c6c84a08b6c0640b186df1eaa72242232faa4cae94b1d6802510), [staging](https://deploy-preview-166--docs-atlas-architecture.netlify.app/industry-solutions/))

Thoughts?

Reply:
Sounds like a good idea to me, added.